presto-orc/src/test/java/com/facebook/presto/orc/OrcReaderTestingUtils.java

```diff
@@ -29,4 +29,13 @@ public static OrcReaderOptions createDefaultTestConfig()
                 new DataSize(1, MEGABYTE),
```

Comment:
Also call this function `createTestingReaderOptions` and it should delegate to the one below with `zstdJniDecompressionEnabled = false`.